src/System.Buffers/ref/System.Buffers.builds

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Buffers.csproj">
```

Comment:
If there is only one reference assembly, then why do we need a .builds file in here? It should be sufficient to just have the .csproj by itself, and have it's NugetTargetMoniker to be set to netstandard1.1